### PR TITLE
Hometask 3 and 4

### PR DESCRIPTION
**Pull request by DenysMiroshnyk**

### CODE REVIEW

**Comment by DenysMiroshnyk** [comment]
Idle has a problem with printing 웃 symbol );

File: Denys_Miroshnyk/3/Crypto Square.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    """
+    normalized_sentence = ''.join(
+        [ch for ch in sentence if ch.isalpha()]).casefold()
+
+    columns = int(sqrt(len(normalized_sentence))) + 1
+    squared_sentence = list()
+
+    for i in range(columns):
+        start = int(columns * i)
+        end = int(min(start + columns, len(normalized_sentence)))
```
Comment (oleksandr-medviediev) [inline Denys_Miroshnyk/3/Crypto Square.py]: min returns int, so you don't need to convert it

File: Denys_Miroshnyk/3/Spell number.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        if speller(recursive_number) != "zero":
+            spelled_number = speller(recursive_number) + \
+                " million " + spelled_number
+
+        recursive_number = ''.join(temp_number[-12:-9])
+        if speller(recursive_number) != "zero":
+            spelled_number = speller(recursive_number) + \
+                " billion " + spelled_number
+        return spelled_number
+
```
Comment (oleksandr-medviediev) [inline Denys_Miroshnyk/3/Spell number.py]: Not the best approach using all these if statements

File: Denys_Miroshnyk/3/Verify brackets.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        bool -- True is brackets are correct
+    """
+
+    bracket_stack = [ch for ch in string if ch in brackets or ch in brackets.values()]
+    bracket_stack.reverse()
+    pair_stack = list()
+
+    while bracket_stack:
+
+        next = bracket_stack.pop()
```
Comment (oleksandr-medviediev) [inline Denys_Miroshnyk/3/Verify brackets.py]: next is a built-in. You can't use it as a name!

File: Denys_Miroshnyk/4/Game_loop.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import Map_generator
```
Comment (oleksandr-medviediev) [inline Denys_Miroshnyk/4/Game_loop.py]: must be named using lower case

File: Denys_Miroshnyk/4/Main.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import Map_generator
+import Game_loop
+
+field_size = 10
+
+while (True):
```
Comment (oleksandr-medviediev) [inline Denys_Miroshnyk/4/Main.py]: You don't need those parentheses

**Pull request merged**